tests/integration/conftest.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for flask charm integration tests."""
+
+import json
+
+import pytest_asyncio
+from pytest_operator.plugin import OpsTest
+
+
+@pytest_asyncio.fixture(scope="module", name="get_unit_ips")
+async def fixture_get_unit_ips(ops_test: OpsTest):
```

Comment:
Out of curiosity, what is the benefit of returning a function to retrieve unit ip addresses rather over returning the unit IP addresses directly?